CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+# Changelog
+
+[npm][1]
+
+[1]: https://www.npmjs.com/package/@google-cloud/vision
+
+## v0.21.0
+
+### Implementation Changes
+
+- Re-generate library using /synth.py (#105)
+- Re-generate library using /synth.py (#100)
+- Re-generate library using /synth.py (#91)
+- chore: Re-generate library using /synth.py (#88)
+
+### Dependencies
+
+- fix: update and cleanup dependencies (#102)
+- fix(deps): update dependency yargs to v12 (#103)
+- chore(package): update eslint to version 5.0.1 (#98)
+- chore(package): update eslint to version 5.0.0 (#90)
+- chore: update sample lockfiles (#95)
+- repo: setup greenkeeper.json to update dependencies in samples/package.json (#94)
+- refactor: drop dependency on extend (#85)
+- fix: move async to dev dependencies (#83)
+- fix: drop support for node 4.x and 9.x (#82)
```

Comment:
This is scary because it is VERY much a breaking change, and needs to be called out as such.  How are you tracking those in pythonlandia?  We have used:
- `semver: major` label on the PR
- `BREAKING CHANGE: ...` in the commit message as laid out in https://conventionalcommits.org/

Reply:
1. We don't make many.
2. We just call it out in the PR title ("Drop 3.4 support - breaking change" or similar) we don't *enforce* any particular style and that's been fine.